scripts/tenderly/deploy-staking-v2.ts

```diff
@@ -116,11 +116,6 @@ async function main() {
         OPTIMISM_PDAO
     );
     await transferOwnership(escrowMigrator, "EscrowMigrator", OPTIMISM_PDAO);
-    await transferOwnership(
-        rewardsNotifier,
-        "StakingRewardsNotifier",
-        OPTIMISM_PDAO
-    );
```

Comment:
did you burn ownership through the script or manually?

Reply:
Yeah the ownership is burnt earlier in the script and hence adding this here was a bug that caused the script to fail, as the `OPTIMISM_PDAO` cannot execute the transaction as by this point `address(0)` is the owner of the contract.